src/Testcontainers.EventStore/EventStoreBuilder.cs

```diff
@@ -0,0 +1,73 @@
+﻿namespace Testcontainers.EventStore
+{
+  public class EventStoreBuilder : ContainerBuilder<EventStoreBuilder, EventStoreContainer, EventStoreConfiguration>
+  {
+    public const string EventStoreImage = "eventstore/eventstore:21.2.0-buster-slim";
+
+    public const ushort EventStorePort = 2113;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="EventStoreBuilder" /> class.
+    /// </summary>
+    public EventStoreBuilder()
+      : this(new EventStoreConfiguration())
+    {
+      DockerResourceConfiguration = Init().DockerResourceConfiguration;
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="EventStoreBuilder" /> class.
+    /// </summary>
+    /// <param name="dockerResourceConfiguration">The Docker resource configuration.</param>
+    private EventStoreBuilder(EventStoreConfiguration dockerResourceConfiguration)
+      : base(dockerResourceConfiguration)
+    {
+      DockerResourceConfiguration = dockerResourceConfiguration;
+    }
+
+    /// <inheritdoc />
+    protected override EventStoreConfiguration DockerResourceConfiguration { get; }
+
+
+    /// <inheritdoc />
+    public override EventStoreContainer Build()
+    {
+      return new EventStoreContainer(DockerResourceConfiguration, TestcontainersSettings.Logger);
+    }
+
+    /// <inheritdoc />
+    protected override EventStoreBuilder Init()
+    {
+      return base.Init()
+        .WithImage(EventStoreImage)
+        .WithPortBinding(EventStorePort, true)
+        .WithEnvironment("EVENTSTORE_CLUSTER_SIZE", "1")
+        .WithEnvironment("EVENTSTORE_RUN_PROJECTIONS", "All")
+        .WithEnvironment("EVENTSTORE_START_STANDARD_PROJECTIONS", "true")
+        .WithEnvironment("EVENTSTORE_EXT_TCP_PORT", "1113")
+        .WithEnvironment("EVENTSTORE_EXT_HTTP_PORT", "2113")
+        .WithEnvironment("EVENTSTORE_INSECURE", "true")
+        .WithEnvironment("EVENTSTORE_ENABLE_EXTERNAL_TCP", "true")
```

Comment:
If TCP port is not bound, does this make sense?

Reply:
Yes, remove env related to TCP, but we need `EVENTSTORE_INSECURE`, without this line fails tests. Base on doc event store, in the new versions, we need to set `EVENTSTORE_INSECURE=true` for security reason.